elliottlib/bzutil.py

```diff
@@ -227,6 +227,13 @@ def component(self):
     def status(self):
         return self.bug.fields.status.name
 
+    @property
+    def security_level(self):
+        try:
+            return self.bug.fields.security
```

Comment:
Curious what this returns when the bug is public, False / doesn't exist?

Reply:
If the bug is public there won't be a security attribute. Code will catch the exception and return None.